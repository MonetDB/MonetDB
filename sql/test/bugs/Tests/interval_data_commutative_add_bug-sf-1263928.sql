select date '2005-08-19' + interval '1' day;
select interval '1' day + date '2005-08-19';
