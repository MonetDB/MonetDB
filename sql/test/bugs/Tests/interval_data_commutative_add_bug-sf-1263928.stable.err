stderr of test 'interval_data_commutative_add_bug-sf-1263928` in directory 'sql/test/bugs` itself:


# 14:00:18 >  
# 14:00:18 >  Mtimeout -timeout 180 Mserver "--config=/ufs/niels/scratch/Linux-x86_64/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/niels/scratch/Linux-x86_64/lib/MonetDB:/ufs/niels/scratch/Linux-x86_64/lib/bin" --set "gdk_dbfarm=/ufs/niels/scratch/Linux-x86_64/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/niels/scratch/Linux-x86_64/var/MonetDB/log" --set mapi_port=31450 --set sql_port=47914 --set xquery_port=53009 --set monet_prompt= --trace "--dbname=mTests_src_test_bugs" --dbinit="module(sql_server); sql_server_start();" ; echo ; echo Over..
# 14:00:18 >  


# 14:00:19 >  
# 14:00:19 >  Mtimeout -timeout 60 MapiClient -lsql -u monetdb -P monetdb --host=localhost --port=47914  < interval_data_commutative_add_bug-sf-1263928.sql
# 14:00:19 >  


# 14:00:19 >  
# 14:00:19 >  Done.
# 14:00:19 >  

