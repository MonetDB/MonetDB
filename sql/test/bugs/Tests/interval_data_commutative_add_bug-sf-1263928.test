query T rowsort
select date '2005-08-19' + interval '1' day
----
2005-08-20

query T rowsort
select interval '1' day + date '2005-08-19'
----
2005-08-20

