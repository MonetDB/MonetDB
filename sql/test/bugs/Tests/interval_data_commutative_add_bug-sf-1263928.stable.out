stdout of test 'interval_data_commutative_add_bug-sf-1263928` in directory 'sql/test/bugs` itself:


# 20:18:59 >  
# 20:18:59 >   mserver5 "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=34282 --set xrpc_port=47106 --set monet_prompt= --trace  "--dbname=mTests_src_test_bugs" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 20:18:59 >  

# MonetDB server v5.9.0, based on kernel v1.27.0
# Serving database 'mTests_src_test_bugs', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://alf.ins.cwi.nl:34282/
# MonetDB/SQL module v2.27.0 loaded

#function user.main():void;
#    clients.quit();
#end main;


# 11:55:47 >  
# 11:55:47 >  Mtimeout -timeout 60 MapiClient -lsql -umonetdb -Pmonetdb --host=localhost --port=35185  < interval_data_commutative_add_bug-sf-1263928.sql
# 11:55:47 >  

#select date '2005-08-19' + interval '1' day;
% .%2 # table_name
% %2 # name
% date # type
% 10 # length
[ 2005-08-20	]
#select interval '1' day + date '2005-08-19';
% .%2 # table_name
% %2 # name
% date # type
% 10 # length
[ 2005-08-20	]

# 12:56:39 >  
# 12:56:39 >  Done.
# 12:56:39 >  

